indicators.el

```diff
@@ -368,15 +368,29 @@ more indicators are on the same physical line."
         indicator))))
 
 (cl-defun ind-remove-indicator-at-line (line)
-  "Remoe the indicator on LINE."
+  "Remove the indicator on LINE."
   (save-excursion
     (goto-char (point-min))
     (forward-line (1- line))
     (ind-remove-indicator (point))))
 
 (cl-defun ind-remove-indicator (pos)
   "Remove the indicator to position POS."
-  (remove-overlays pos pos 'ind-indicator-absolute t))
+  (save-excursion
+    (goto-char pos)
+    (delete-dups ind-managed-absolute-indicators)
```

Comment:
Where is this function coming from?

Reply:
This is from `subr.el` file.

Reply:
Ah okay, I should've checked.  `subr.el` is included automatically so you don't need to require.